### ISSUE TITLE
Sweep Control cannot call Continuous B Sweep

### ISSUE THREAD

**Issue by ciozi137** [levylabpitt/Transport#71]
#### Issue
Bottom line:**Sweep Control.vi* cannot take data **_while_** a parameter is being swept (temperature, magnetic field, etc).

**Continuous B Sweep.vi** accomplishes the above task by calling "Take Measurement.vi" in parallel with "Set B.vi" and "Get B.vi". When the target is reached the "Take Measurement" loop is terminated via an event/notifier.

Sweep Control.vi cannot call Continuous B Sweep.vi because they both call Take Measurement, resulting in reentrancy error.

#### Options to Fix
- Give continuous sweep capabilities to Sweep Control (and deprecate Continuous B Sweep)
  - Doing this may easily enable other continuous variables (temperature, etc)
- Generalize Sweep/Sweep Parameter capabilities into an SMO architecture 
- Generalize into Python

**Comment by ciozi137** [comment]
This is being put on hold, perhaps indefinitely while we transition to a python-based sequencer
